lms/templates/wiki/includes/editor.html

```diff
@@ -9,34 +9,3 @@
   });
 </script>
 </div>
-
-<div>
```

Comment:
As far as I can tell, there is never a "plugin" in "sidebar" with legitimate values. On production, this causes this HTML to be injected but without any text in the h2 element, and also without a legitimate href. This causes accessibility errors.

![image](https://cloud.githubusercontent.com/assets/484484/22484804/fc6fe2d8-e7d0-11e6-957d-60b7e8282cff.png)